### ISSUE TITLE
ipywidgets 7.6.2 does not install with Python 3.5

### ISSUE THREAD

**Issue by patviafore-canonical** [jupyter-widgets/ipywidgets#3058]
### Problem
When trying to install ipywidgets in a Python 3.5 environment, I get the following error:

 pip install ipywidgets
 ... snip  ....
 Collecting jupyterlab-widgets>=1.0.0; python_version >= "3.5" (from ipywidgets)
  **Could not find a version that satisfies the requirement jupyterlab-widgets>=1.0.0; python_version >= "3.5" (from ipywidgets) (from versions: 0.3.2, 0.3.3, 0.4.0, 0.4.1, 0.5.0, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.6.7, 0.6.8, 0.6.9, 0.6.10, 0.6.11, 0.6.12, 0.6.13, 0.6.14, 0.6.15)**
No matching distribution found for jupyterlab-widgets>=1.0.0; python_version >= "3.5" (from ipywidgets)

Jupyterlab widgets 1.0.0 is 3.6 only according to https://pypi.org/project/jupyterlab-widgets/. I confirmed this with a download of jupyterlab widgets from pypi.

### Proposed Solution
* Restrict jupyterlab_widgets to only be installed when 3.6 is installed.

### Additional context

What's odd about this is that the GitHub source for ipywidgets does not match up with what's downloaded from Pypi for 7.6.2 (I can't find the source code for 7.6.2 reflected in GitHub. If you could point me to where that code lives, I'd be happy to submit a PR.

It might also be that the upcoming 8.0.0 release covers this, in which case I'm okay marking this as resolved, but since source code didn't match what's in pypi, I thought I'd check. 

For reference, in setup.py, this is the offender:
```
extras_require = setuptools_args['extras_require'] = {
    ':python_version<"3.3"' : ['ipython>=4.0.0,<6.0.0'],
    ':python_version>="3.3"': ['ipython>=4.0.0'],
    ':python_version>="3.5"': ['jupyterlab_widgets>=1.0.0'],
    'test:python_version=="2.7"': ['mock'],
    'test': ['pytest>=3.6.0', 'pytest-cov'],
}
```

**Comment by jasongrout** [comment]
Yes, please submit a PR to the 7.x branch. Thanks!

**Comment by jasongrout** [comment]
Fixed by #3059, released as ipywidgets 7.6.3 (which I verified does install in python 3.5). Thanks again!
